### ISSUE TITLE
[Request] Direct Only Fill-in

### ISSUE THREAD

**Issue by na7q** [markqvist/MicroDigi#2]
Would it be possible to make a direct only fill in mode?
One that only responds to stations not already digipeated. It would be more useful than the old fill-in method.

**Comment by markqvist** [comment]
I'm not sure I understand it completely. Do you mean that it waits for a while to see if another digi repeats the packet, and if not, it will digipeat it? Or that it will only digipeat packets directed to it's callsign?

**Comment by markqvist** [comment]
By the way, this firmware is very unfinished (although it does work), but I am thinking of putting the finishing work in it, and getting it tested. Do you run any digi sites where it could be tested out?

**Comment by na7q** [comment]
Sorry for the late reply. Busy work week. 
What I mean, is that it only digipeats stations heard directly and not already digipeated. 
So if it hears the station direct, it digipeats them. If it hears through a digipeater, and not direct, it does not digipeat. Hope that makes sense. I run 4 digipeaters at various locations, I could certainly test it! And would be more than willing.

**Comment by markqvist** [comment]
Yes, that makes sense. I could certainly implement that. I'll put it on my list, but it'll be another few weeks before I get around to this, I'm pretty busy over in the https://github.com/markqvist/reticulum repository at the moment :)

**Comment by LU7HQW** [comment]
Hello markqvist. I'm very glad to see you are going to improve the digipeater firmware. I'm going to put in service two digipeaters with your firmware,  very soon. And i really want to collaborate testing the new versions. 73's de Aldo, LU7HQW.
